src/Uno.UI/UI/Xaml/ApplicationActivity.Android.cs

```diff
@@ -371,6 +371,16 @@ protected override void OnActivityResult(int requestCode, Result resultCode, Int
 			}
 		}
 
+		protected override void AttachBaseContext(Context @base)
+		{
+			if (FeatureConfiguration.Font.IgnoreTextScaleFactor && @base?.Resources?.Configuration is { } config)
+			{
+				config.FontScale = 1.0f;
```

Comment:
Interesting trick! What does testing in the sample app give you (using the PR artifacts)?